mentorApp/lib/main.dart

```diff
@@ -15,8 +17,10 @@ class App extends StatelessWidget {
       // When navigating to the "/" route, build the FirstScreen widget.
       '/': (context) => FirstRoute(),
       // When navigating to the "/second" route, build the SecondScreen widget.
-      '/second': (context) => SecondRoute(),
```

Comment:
Usually, the page navigations are lowercased. So it's a good idea to keep it as `'/second'` and `'/user-profile'`.